packages/angular_devkit/build_angular/src/differential-loading/update-webpack-config.ts

```diff
@@ -0,0 +1,132 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+type WebpackConfig = any; // tslint:disable-line
+type WebpackPlugin = any; // tslint:disable-line
+
+export function updateWebpackConfig(config: WebpackConfig): WebpackConfig {
```

Comment:
"update" is not descriptive. could you find a name that says what is the update?

createSubConfigs or something? (what does webpack call it when configs are nested in this way?)